cost/kubecost/sizing/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## v0.3
+
+- A new parameter for requesting recommendations "per cluster" or "overall" was added, if per cluster mode is configured, policy will discover clusters, iterate the clusters and request recommendations for each one. Previously, policy requested these recommendations in overall mode.
```

Comment:
Since "per cluster" and "overall" are options probably you can put them into "" everytime, since you used it just first time.

Reply:
@juanestrahcl - You are right, I put those between quotes